editors/code/package.json

```
@@ -1,16 +1,28 @@
 {
 	"name": "air",
 	"displayName": "Air",
-	"description": "Air Language Server",
+	"description": "R formatter and language server for Visual Studio Code",
```

Since that will show up in Positron too

```suggestion
	"description": "R formatter and language server",
```